src/sentry/static/sentry/app/views/groupEvents.jsx

```diff
@@ -82,9 +82,11 @@ const GroupEvents = React.createClass({
           pageLinks: jqXHR.getResponseHeader('Link')
         });
       },
-      error: (error) => {
+      error: (err) => {
+        let error = err.responseJSON || true;
+        error = error.detail || true;
```

Comment:
sorry, don't follow - do you mean you would have expected true.responseJSON to throw?